### ISSUE TITLE
Graph does not arrange according to available size

### ISSUE THREAD

**Issue by ido-ran** [thinkpixellab/bot#33]
I'm pretty sure that on one of the old versions of the Graph control when I change the size of the window, and by that changing the available size of the Graph control, the children were re-arrange to use the available size.
The new version does not do that, it arrange the children as if it had all the space it require.

Thank you,
Ido.

**Comment by kevmoo** [comment]
Good idea. Feel free to take a crack at it if you like. Not sure I'll get to this any time soon, but I'd love a pull request.
